File: 2015/fall/psets/2/hacker2/generator.c
#define _XOPEN_SOURCE
#include <stdio.h>
#include <unistd.h>

int main(void)
{
    printf("andi:%s\n", crypt("Berkeley", "HA"));
    printf("caesar:%s\n", crypt("13", "50"));
    printf("eli:%s\n", crypt("yale", "50"));
    printf("hdan:%s\n", crypt("boola", "50"));
    printf("jason:%s\n", crypt("Silliman", "50"));
    printf("john:%s\n", crypt("harvard", "50"));
    printf("levatich:%s\n", crypt("PBJ", "50"));
    printf("rob:%s\n", crypt("password", "50"));
    printf("skroob:%s\n", crypt("12345", "50"));
    printf("zamyla:%s\n", crypt("password", "HA"));
}

File: 2015/fall/psets/7/pset7/generator.php
#!/bin/env php
<?php

     printf("INSERT INTO `users` (username, hash) VALUES('%s', '%s');\n", "andi", password_hash("Berkeley", PASSWORD_DEFAULT));
     printf("INSERT INTO `users` (username, hash) VALUES('%s', '%s');\n", "caesar", password_hash("13", PASSWORD_DEFAULT));
     printf("INSERT INTO `users` (username, hash) VALUES('%s', '%s');\n", "eli", password_hash("yale", PASSWORD_DEFAULT));
     printf("INSERT INTO `users` (username, hash) VALUES('%s', '%s');\n", "hdan", password_hash("boola", PASSWORD_DEFAULT));
     printf("INSERT INTO `users` (username, hash) VALUES('%s', '%s');\n", "jason", password_hash("Silliman", PASSWORD_DEFAULT));
     printf("INSERT INTO `users` (username, hash) VALUES('%s', '%s');\n", "john", password_hash("harvard", PASSWORD_DEFAULT));
     printf("INSERT INTO `users` (username, hash) VALUES('%s', '%s');\n", "levatich", password_hash("PBJ", PASSWORD_DEFAULT));
     printf("INSERT INTO `users` (username, hash) VALUES('%s', '%s');\n", "rob", password_hash("password", PASSWORD_DEFAULT));
     printf("INSERT INTO `users` (username, hash) VALUES('%s', '%s');\n", "skroob", password_hash("12345", PASSWORD_DEFAULT));
     printf("INSERT INTO `users` (username, hash) VALUES('%s', '%s');\n", "zamyla", password_hash("password", PASSWORD_DEFAULT));

?>
